en/docs/services/create-your-first-service.md

```diff
@@ -1,168 +1,161 @@
 
-    
 Now you have completed designing your service. It looks as follows.
 
-![No Code View](../assets/img/services/choreo-service-low-code-view.png)
+- In the low-code view
 
-The code view looks as follows.
+   ![No Code View](../assets/img/services/choreo-service-low-code-view.png)
 
-![Code View](../assets/img/services/choreo-service-code-view.png)
+- In the code view
 
+   ![Code View](../assets/img/services/choreo-service-code-view.png)
     
 To validate the service, click **Run & Test**. The following is logged to indicate that you have successfully started the service.
 
-![Service Started Log](../assets/img/services/service-started-notification.png)
+ ![Service Started Log](../assets/img/services/service-started-notification.png)
     
 Congratulations! You have successfully created your first service.
    
 ## Step 2: Test the service
 
-To test the `covid-stats` service you created, follow the procedure below:
+To test the `covid-stats` service you created, follow this procedure:
 
 1. Click the **Test** icon in the left pane.
 
-    ![Test Icon](../assets/img/services/test-icon.png)
+   ![Test Icon](../assets/img/services/test-icon.png)
 
 2. In the section that opens to the right of the page, click **GET**.
 
 3. Click **Try it out**, and then click **Execute**.
 
-The search results for COVID-19 statistics of the United States are displayed as the server response as shown below.
+The following figure shows the server response to the search results for the United States COVID-19 statistics:
 
 ![Server Response](../assets/img/services/server-response.png)
 
-Now you have verified that the `covid-stats` service works as expected. Therefore, you can deploy it.
+Now that you have verified that the **covid-stats** service works as expected, you can deploy it.
 
 ## Step 3: Deploy the service
 
-To deploy the `covid-stats` service, follow the procedure below:
+To deploy the `covid-stats` service, follow this procedure:
 
 1. Click the **Go Live** icon in the left pane.
 
-    ![Test Icon](../assets/img/services/deploy-icon.png)
+      ![Test Icon](../assets/img/services/deploy-icon.png)
 
 2. To deploy the service, click **Deploy**.
 
-    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    
+    The status of the service changes to **Deployed**, and the following logs show that the product is successfully deployed:    
 
-    ![Deployment Logs](../assets/img/services/deployment-logs.png)
+      ![Deployment Logs](../assets/img/services/deployment-logs.png)
 
-3. Once the service is deployed, click on the **Go Live** tab below the low-code design view. Then click **://cURL** and copy the cURL command that is displayed.
+3. Once the service is deployed, click the **Go Live** tab below the low-code diagram. Then click **://cURL** and copy the cURL command that is displayed.
```

Comment:
Shall we re-phrase to " Then click **://cURL** and copy the cURL command"

Reply:
+1
Restructured the content via https://github.com/wso2/docs-choreo-dev/pull/108.